Client/Frontend/Browser/BrowserViewController/OpenSearch/BrowserViewController+OpenSearch.swift

```diff
@@ -190,13 +190,20 @@ extension BrowserViewController {
     }
     
     private func createSearchEngine(_ url: URL, reference: String, icon: UIImage) {
-        NetworkManager().downloadResource(with: url).uponQueue(.main) { [weak self] response in
-            guard let openSearchEngine = OpenSearchParser(pluginMode: true).parse(
-                    response.data, referenceURL: reference, image: icon, isCustomEngine: true) else {
-                return
+        Task.detached(priority: .userInitiated) {
```

Comment:
What's the reason for detaching the task here? `downloadResource` will be running network tasks in a detached task anyways